src/main/kotlin/platform/mixin/handlers/injectionPoint/AtResolver.kt

```diff
@@ -68,8 +74,13 @@ class AtResolver(
     private val targetMethod: MethodNode,
 ) {
     companion object {
+        private val injectionPointAnnotations = mapOf(
+            CONSTANT to "CONSTANT"
+        )
```

Comment:
I'm already thinking this could be an extension point

Reply:
It wouldn't really have an implementation class? I agree it should probably be extensible but I'm not sure exactly how to approach that

Reply:
Extension points don't need an implementation class. You can create a bean class which contains the annotation name and the `atCode`, and specify the extension point and the bean class in the `plugin.xml`. Your EP name would be a `ExtensionPointName<YourBeanClass>`, which is essentially a list of `YourBeanClass`. If you're worried about performance, you can create a map from that and update it dynamically using an extension point listener.

Reply:
Also maybe rename this to `impliedInjectionPoints` or something along those lines.

Reply:
Ah, gotcha. I misunderstood the point of the bean class.

Reply:
Something like `annotationAliasForInjectionPoint` maybe? It's kinda hard to explain succinctly

Reply:
On second thought, I think `injectionPointAnnotation` is fine.